src/handlers/RF24StreamLogHandler.cpp

```diff
@@ -141,73 +151,47 @@ void RF24StreamLogHandler::appendFormat(const char format, va_list *args)
 void RF24StreamLogHandler::appendTimestamp()
 {
     char c[12];
-    sprintf(c, "%10lu ", millis());
+    sprintf(c, "%10lu", millis());
     stream->print(c);
-    stream->print(" ");
 }
 
 void RF24StreamLogHandler::appendLogLevel(uint8_t logLevel)
 {
-    uint8_t logMainLevel = logLevel & 0xF8;
     uint8_t subLevel = logLevel & 0x07;
 
-    switch (logMainLevel)
+    if (logLevel >= RF24LogLevel::ERROR && logLevel <= RF24LogLevel::DEBUG + 7)

```

Comment:
@2bndy5, I just have reworked your condition. This one look easier - at least for me. It checks if the log level is from a "valid" main log level groups (ERROR, WARN, INFO, DEBUG).

Reply:
looks like the same result is reached, so I have no problems.